src/Analysis/Ast/Impl/Resources.resx

```diff
@@ -189,4 +189,7 @@
   <data name="UnableToDetermineCachePathException" xml:space="preserve">
     <value>Unable to determine analysis cache path. Exception: {0}. Using default '{1}'.</value>
   </data>
+  <data name="UnsupporedOperandType" xml:space="preserve">
+    <value>Unsupported operand type(s) for '{0}': '{1}' and '{2}'</value>
```

Comment:
There are always two types listed, so you can just write `types` instead of `type(s)`.

Reply:
Ok!